images.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"github.com/bwmarrin/discordgo"
+)
+
+
+
+func init() {
+	registerCommand("erasmus", "links", "Link naar erasmus", sayErasmus)
+}
+
+func sayErasmus(s *discordgo.Session, m *discordgo.MessageCreate) {
+	embed := NewEmbed()
+	embed.SetTitle("Erasmus @ ITfactory")
+	embed.SetImage("https://static.eyskens.me/thomas-bot/sem_2_2020.gif")
+	embed.SetURL("https://thomasmore365.sharepoint.com/sites/james/NL/international?tmbaseCampus=Geel")
+	s.ChannelMessageSendEmbed(m.ChannelID, embed.MessageEmbed)
+}
+
+func init() {
+	registerCommand("partners", "links", "Link naar partners", sayPartners)
+}
+
+func sayPartners(s *discordgo.Session, m *discordgo.MessageCreate) {
+	embed := NewEmbed()
+	embed.SetTitle("Partners in education")
+	embed.SetImage("https://static.eyskens.me/thomas-bot/voorstelling_partners_in_education.png")
```

Comment:
@AnnHannes hebben we hier een linkje bij dat je naar meer info brengt?

Reply:
@meyskens  
![image](https://user-images.githubusercontent.com/57891540/78115518-3324e000-7403-11ea-831e-28a519414be9.png)